src/test/java/subway/acceptance/LineRequestFixture.java

```diff
@@ -0,0 +1,65 @@
+package subway.acceptance;
+
+import io.restassured.RestAssured;
+import io.restassured.response.ExtractableResponse;
+import io.restassured.response.Response;
+import org.springframework.http.MediaType;
+
+import java.util.HashMap;
+import java.util.Map;
+
+public class LineRequestFixture {
+
+    public static ExtractableResponse<Response> 지하철_노선_생성(String name, String color, Long upStationId, Long downStationId, Long distance) {
+        Map<String, String> params = new HashMap<>();
+        params.put("name", name);
+        params.put("color", color);
+        params.put("upStationId", upStationId.toString());
+        params.put("downStationId", downStationId.toString());
+        params.put("distance", distance.toString());
+
+        ExtractableResponse<Response> response =
+                RestAssured.given().log().all()
+                        .body(params)
+                        .contentType(MediaType.APPLICATION_JSON_VALUE)
+                        .when().post("/lines")
+                        .then().log().all()
+                        .extract();
+        return response;
```

Comment:
```suggestion
        assertThat(response.statusCode()).isEqualTo(HttpStatus.CREATED.value());
        return response;
```

static 메서드 사이에 HttpCode에 대한 검증 로직을 추가하는 건 어떨까요~?